src/Aquarius.ONE.ClientSDK/Common/Activity/ActivityApi.cs

```diff
@@ -191,8 +192,11 @@ public async Task<bool> UpdateActivitiesAsync(Proto.Activities activities)
 
             try
             {
+                var endpoint = "/common/activity/v1";
+                if (updatePropertyBag)
+                    endpoint += "?updatePropertyBag=true";
                 var json = JsonConvert.SerializeObject(activities, _serializerSettings);
-                var respContent = await _restHelper.PutRestJSONAsync(Guid.NewGuid(), json, "/common/activity/v1/").ConfigureAwait(_continueOnCapturedContext);
+                var respContent = await _restHelper.PutRestJSONAsync(Guid.NewGuid(), json, endpoint).ConfigureAwait(_continueOnCapturedContext);

```

Comment:
Do we have any change to Claros.Common.Activity PUT endpoint for adding this new query parameter?  Or are they incoming.

Reply:
The updatePropertyBag query parameter is already included in the Activity and Schedule MSApps.